src/scripts/main.js

```diff
@@ -354,7 +354,48 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const newRow = document.createElement('tr');
+  table.append(newRow);
+
+  function sexChecking(value) {
+    if (value === 'm') {
+      return 'Male';
+    }
+
+    if (value === 'f') {
+      return 'Female';
+    }
+  }
+
+  const newNameInfo = document.createElement('td');
+  newNameInfo.textContent = person.name;
+  newRow.append(newNameInfo);
+
+  const newGendernfo = document.createElement('td');
```

Comment:
There seems to be a typo in the variable name `newGendernfo`. It should likely be `newGenderInfo` to maintain consistency with other variable names.